scikit-multiflow/src/skmultiflow/prototype/robust_soft_learning_vector_quantization.py

```diff
@@ -1,407 +0,0 @@
-import math
```

Comment:
You guys don't have to remove the files from the `skmultiflow` directory, we'll remove it entirely once we merge anyway :)

Reply:
This is to keep the history of the file (`git mv`). Also helps to clearly highlight what we are modifying. Makes things easier to track down issues from the required changes and (minor) maintenance that we are performing.